spec/spec_helper.rb

```diff
@@ -5,6 +8,8 @@
 require "webmock/rspec"
 require "pry"
 
+WebMock.disable_net_connect!(allow_localhost: true, allow: 'codeclimate.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.